src/UsedSymbolExtractor.php

```diff
@@ -272,14 +248,12 @@ public function parseUseStatement(): ?array
             } elseif ($token === '{') {
                 $groupRoot = $class;
                 $class = '';
-            } elseif ($token === '}') {
-                continue;
-            } else {
+            } elseif ($token === '(') {
```

Comment:
I think termination on "other" token (previous logic) is more readable than this. So if adding whitespace tokens here can accomplish that, I think we should do it.